core/core_modules/help.js

```diff
@@ -11,47 +13,41 @@ constructHelpMessage = function(help, modules, context, prefix) {
 },
 
 checkIfModuleExists = function(modules, moduleName) {
-	return modules.find(function(element, index, array) {
+	return modules.find(function(element) {
 		return element.name === moduleName;
 	});
 },
 
-shortSummary = function(prefix) {
+shortSummary = function(context) {
 	var help = figlet.textSync(this.packageInfo.name.toProperCase()) + '\n '
 		+ this.packageInfo.version + '\n--------------------\n'
 		+ this.packageInfo.homepage +  '\n\n';
 
-	/* Deprecated */
-	var context = {
-		commandPrefix: prefix
-	};
-
-	help = constructHelpMessage(help, this.coreModules, context, prefix);
-	return constructHelpMessage(help, this.loadedModules, context, prefix);
+	help = constructHelpMessage(help, this.coreModules, context);
+    return constructHelpMessage(help, this.loadedModules, context);
 },
 
-longDescription = function (moduleName, prefix) {
+longDescription = function(moduleName, context) {
 	var module = checkIfModuleExists(this.coreModules, moduleName);
 
 	if (!module || module.length === 0) {
 		// Check loaded modules, as commnd not in core modules
 		module = checkIfModuleExists(this.loadedModules, moduleName);
 	}
-	
+
 	if (!module || module.length === 0) {
 		return 'Cannot provide help on module that was not found. Has it been disabled?';
 	}
-	
+
 	if (module.length > 1) {
 		return 'More than one module has the same name. Please fix this before continuing.';
 	}
 
 	var help = '',
-		context = {
-			commandPrefix: prefix
-		},
-		cmdHelp = module.help.call(context, prefix);
-		
+        cmdHelp = module.ignoreHelpContext
+            ? module.help(context.commandPrefix)
+            : module.help.call(context, context.commandPrefix);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [':' should be placed at the end of the line.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530333283/issues/source?bid=3282987&fileBranchId=3282997#l49)